source/src/cip/ciptypes.h

```diff
@@ -191,9 +191,7 @@ typedef struct {
  *
  */
 typedef struct {
-  EipUint8 path_size;
-  /**< Size of the Path in 16-bit words *//* TODO: Fix, should be UINT
-   (EIP_UINT16) */
+  EipUint16 path_size; /**< Size of the Path in 16-bit words */
   EipUint16 class_id; /**< Class ID of the linked object */
   EipUint16 instance_number; /**< Requested Instance Number of the linked object */
```

Comment:
If 32 bit instance number shall be supported, this variable should also be changed to EipUint32